src/main/java/org/hawkular/client/android/backend/BackendClient.java

```diff
@@ -136,14 +136,15 @@ public void configureCommunication(@NonNull String host,
 
     private void configurePipes(URL backendUrl, Persona persona) {
         URL pipeUrl = Urls.getUrl(backendUrl, BackendPipes.Paths.ROOT);
-
+        URL alertResolveUrl = Urls.getUrl(pipeUrl, BackendPipes.Paths.ALERT_RESOLVE);
+        URL alertAckUrl = Urls.getUrl(pipeUrl, BackendPipes.Paths.ALERT_ACKNOWLEDGE);
         List<PipeModule> pipeModules = Arrays.asList(
             getAuthorizationModule(),
             getPersonnelModule(persona));
 
         configurePipe(BackendPipes.Names.ALERTS, pipeUrl, pipeModules, Alert.class);
-        configurePipe(BackendPipes.Names.ALERT_ACKNOWLEDGE, pipeUrl, pipeModules, String.class);
-        configurePipe(BackendPipes.Names.ALERT_RESOLVE, pipeUrl, pipeModules, String.class);
+        configurePipe(BackendPipes.Names.ALERT_ACKNOWLEDGE, alertAckUrl, pipeModules, String.class);
```

Comment:
You previous change lead to a little confusion here my friend. `a34f0421` didn’t consider that we have a single `ROOT` path. So, you should do this instead:

```java
// BackendClient.java

configurePipe(BackendPipes.Names.ALERT_ACKNOWLEDGE, pipeUrl, pipeModules, String.class);
configurePipe(BackendPipes.Names.ALERT_RESOLVE, pipeUrl, pipeModules, String.class);
...
configurePipe(BackendPipes.Names.PERSONA, pipeUrl, pipeModules, Persona.class);
configurePipe(BackendPipes.Names.PERSONAS, pipeUrl, pipeModules, Persona.class);
```

```java
// BackendPipes.java

public static final String PERSONA = "accounts/personas/current";
public static final String PERSONAS = "accounts/personas";
```

And then you can remove your introduced variables. This should be much better.